charts/pwa/values.schema.json

```diff
@@ -6,11 +6,20 @@
     "description": "The Helm Values Schema to define values necessary to deploy Intershops PWA.",
     "required": [
         "upstream",
-        "hybrid",
```

Comment:
Please check that the conditional dependency of pwa `chart.yaml` is not doing something wrong when `hybrid` object is missing in the `values.yaml`

Reply:
it looks like `{{ .Values.hybrid.enabled }}` is just evaluated to false without warning or error, if the `hybrid` element is missing, so I guess we are good.